source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -0,0 +1,284 @@
+   select :guilabel:`Name Partition`.  
+
+#. Set the name to “CLR_BOOT”.  See Figure 6 and Figure 7.  
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-linux-06.png
+      :scale: 100%
+      :alt: Name CLR_BOOT partition
+
+      Figure 6: Name CLR_BOOT partition
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-linux-07.png
+      :scale: 100%
+      :alt: Resulting CLR_BOOT main screen
+
+      Figure 7: Resulting CLR_BOOT main screen
+
+#. Create a swap partition for |CL|.
+
+   a. Right-click the “unallocated” partition and select :guilabel:`New` 
+      to add a new partition.
+
+   #. Enter “250” in the :guilabel:`New Size (MiB)` field to create a 
+      250MB swap space. 
+
+   #. Enter “CLR_SWAP” in the :guilabel:`Partition name` field and select
+      “linux-swap” as the “File system” type.  
+
+   #. Click the “Add” button.  See Figure 8.
+
+      .. figure:: ../../_figures/multi-boot/dual-boot-linux-08.png
+         :scale: 100%
+         :alt: Create CLR_SWAP partition
+
+         Figure 8: Create CLR_SWAP partition
+
+#. Create |CL| root partition.
+
+   a. Right-click “unallocated” partition again and select :guilabel:`New` 
+      to add a new partition.
+
+   #. Create a partition that is at least 21GB, enter “CLR_ROOT” in the 
+      :guilabel:`Partition name` field, and select a :guilabel:`File system` 
+      type of your choice.    
+
+   #. Click the “Add” button.  See Figure 9. 
+
+      .. figure:: ../../_figures/multi-boot/dual-boot-linux-09.png
+         :scale: 100%
+         :alt: Create CLR_ROOT partition
+
+         Figure 9: Create CLR_ROOT partition
+
+#. Click the green checkmark button to create the newly-defined partitions. 
+   See Figure 10.  
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-linux-10.png
+      :scale: 100%
+      :alt: Partitions to be created
+
+      Figure 10: Partitions to be created
+
+#. Close the GParted window, and the |CL| installer will reappear with 
+   the newly-defined partitions to use.  See Figure 11.
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-linux-11.png
+      :scale: 100%
+      :alt: |CL| installer partitions defined
+
+      Figure 11: |CL| installer partitions defined
+
+#. Complete the remaining steps of :guilabel:`Required Options` to 
+   to install |CL|. Complete any :guilabel:`Advanced Options` as desired.
+    
+Ways to boot |CL|
+*****************
+
+Although we installed |CL| last, Ubuntu is still the default boot OS.  There are several ways to boot |CL|:
+
+#. Use your BIOS “Boot Menu” to select and boot |CL|.  
+   This is temporary and will not make |CL| the default boot OS.  
+
+#. Make systemd-boot, the boot loader that |CL| uses, the default
+   boot loader to boot |CL| and chain-boot GRUB; therefore, boot 
+   Ubuntu. Follow the next section to implement this.  
+
+Change default bootloader, chain-boot GRUB
+******************************************
+
+#. Make systemd-boot the Default Bootloader and chain-boot GRUB.
```

Comment:
I would prefer this line to be the section header.  If not, go ahead and deprecate it.  Doesn't make sense for it to be Step #1.  This whole section is about making systemd-boot the default bootloader and chain-booting GRUB.

Reply:
Sure. This was a misunderstanding based on original draft.